### ISSUE TITLE
tdm does not run under windows 11

### ISSUE THREAD

**Issue by mickym2** [jziolkowski/tdm#193]
tdm is connecting to the mqtt broker - as it says that it is connected and log confirms connection. However the window remains empty.
Same config under windows 10 is working properly.

**Comment by DavidTre07** [comment]
Works well on windows 11 on my side.
Connected to mqtt and I'm able to see my devices.

So not sure the problem is related to windows 11.

Did you check in the mqtt broker if some messages are sent/received with windows 11 ? (mosquitto_sub -v -t ...)

**Comment by mickym2** [comment]
I can definitely exclude connection problems. I run a local NodeRed Installation under Windows 11 and connect the mosquitto broker. I can install a virtual machine with tasmota device manager under Windows 10 and can connect the broker and I can also connect with tasmota device manager with Windows 11 .

![image](https://user-images.githubusercontent.com/58858526/172076579-6ac2caeb-5d32-46b7-81af-fc8bb263a991.png)

so the main window remains blank - even if the tdm says that it is connected. Running under Windows 10 in a virtual machine with the exact same settings and therefore the identical communication to the mosquitto broker works:

![image](https://user-images.githubusercontent.com/58858526/172077260-aea2c180-a5ca-4c10-9e3c-1f3dff7cd7f0.png)


On left side tdm with Windows 10 in a virtual machine with max. FHD resolution - left running tdm native on Win11 with 4K resolution.


So it is either a display problem - I tried to reduce the screen resolution as it is a 4K display - but even with FHD (1920x1020) I cant get a display screen. 

So either the tdm can not handle the scaling factor of 200 ( I had similar effects with an other tool) - or the resulation - or it is something else - as I already tried it with a lower resolution.

![image](https://user-images.githubusercontent.com/58858526/172077049-20efa35f-adc7-4b01-b8b1-515bf22757dc.png)
